### ISSUE TITLE
Можно ли отправлять запросы в influxDB 2?

### ISSUE THREAD

**Issue by svetoshkin** [speakyourcode/grafana-button-panel#13]
Добрый день, подскажите, пожалуйста, можно ли отправлять запросы в influxDB 2? и подскажите боди, как написать?

**Comment by speakyourcode** [comment]
The plugin only forwards the queries to the datasources, so if they support InfluxDB 2, so does the plugin

**Comment by fercasjr** [comment]
> The plugin only forwards the queries to the datasources, so if they support InfluxDB 2, so does the plugin

It did not work for me as it is. needed to change url:"api/tsdb/query" to url:"api/ds/query"

**Comment by sviatoshkin** [comment]
Is it possible to add InfluxDB 2 support?

**Comment by fercasjr** [comment]
Yes, I was messing around with the code and changed the above line to make it work, however Grafana won't let you use a tweaked plugin because needs to be signed, however you can enable debug mode and disable plugin signature to run it, or submit the plugin and sign it.
